messageBundlesCompiler.js

```diff
@@ -0,0 +1,198 @@
+/**
+ * The message bundles compiler.
+ *
+ * Contains various functions for compiling a mega message bundle which contains
+ * all messages for all available locales.
+ * Copyright 2017 Raising the Floor - International
+ *
+ * Licensed under the New BSD license. You may not use this file except in
+ * compliance with this License.
+ * The research leading to these results has received funding from the European Union's
```

Comment:
This section of the boilerplate is no longer needed since the relevant project has ended